core/src/main/codegen/templates/Parser.jj

```diff
@@ -243,6 +243,17 @@ public class ${parser.class} extends SqlAbstractParserImpl
         return SqlStmtList();
     }
 
+    public SqlNode parseArray() throws SqlParseException {
+        switchTo(LexicalState.BQID);
+        try {
+          return ArrayLiteral();
+        } catch (ParseException ex) {
```

Comment:
I used catch twice because javaCC can not parse `catch (ParseException | TokenMsgError ex)` for some reason.